src/ui/ResultLayout/ResultLayout.ts

```diff
@@ -37,6 +39,7 @@ export class ResultLayout extends Component {
    * The possible valid and supported layout.
```

Comment:
Actually, the "ValidLayout" type itself is already documented (see lines 21-23) and it automatically shows the possible values in the generated doc site. So documenting the "validLayouts" static attribute seems redundant to me. What do you think?